lib/teleterm/api/uri/uri_test.go

```diff
@@ -43,12 +45,47 @@ func TestURI(t *testing.T) {
 		},
 	}
 
-	for _, tt := range testCases {
+	for _, tt := range tests {
+		tt := tt
 		t.Run(fmt.Sprintf("%v", tt.in), func(t *testing.T) {
+			t.Parallel()
+
 			out := tt.in.String()
-			if !reflect.DeepEqual(out, tt.out) {
-				t.Errorf("out %#v, want %#v", out, tt.out)
-			}
+			require.Equal(t, tt.out, out)
+		})
+	}
+}
+
+func TestParseClusterURI(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		in  uri.ResourceURI
```

Comment:
It feels a little odd that a test that is testing a function that "parses a string and returns a cluster URI", has the input of the test case as a `ResourceURI` and the expected output of the test as a string (it feels reversed!)

This arrangement makes it much harder to add cases to the test in future where we expect an error to be yielded.

Reply:
Yeah, you're right. I was too lazy to type out the actual URIs so I just used the package methods to make them. I replaced them with strings, though I could also just change the types and then slap `.String()` at the end of the values for inputs.

Reply:
Thank you ! That's much better :)